src/modules/evas/engines/software_generic/filters/evas_filter_bump.c

```diff
@@ -103,9 +103,13 @@ _bump_map_cpu_alpha_alpha(Evas_Filter_Command *cmd)
    h = cmd->input->h;
    EINA_SAFETY_ON_FALSE_RETURN_VAL(w > 2 && h > 2, EINA_FALSE);
 
+#define E_READ 0
+#define E_WRITE 0
```

Comment:
Shouldn't this be defined elsewhere (probably at `src/lib/evas/filters/evas_filter_private.h:310` as an `#else` to the `#ifdef HAVE_ECTOR` at line 308) ? Like this we will overwrite any previous definition, won't we?

Reply:
Actually these files aren't even compiled without ector, hm...